js/src/features/project/types.d.ts

```diff
@@ -2,102 +2,127 @@ declare module "ProjectModels" {
   export type ProjectStatusTypeResponse = "Active" | "Closed" | "Suspended";
```

Comment:
I think we should separate domain type from request and response types

Reply:
Feel free to suggest any options

Reply:
Done